src/Controls/src/Core/Animation.cs

```diff
@@ -2,29 +2,45 @@
 using System.Collections;
 using System.Collections.Generic;
 using BaseAnimation = Microsoft.Maui.Animations.Animation;
+
 namespace Microsoft.Maui.Controls
 {
-	/// <include file="../../docs/Microsoft.Maui.Controls/Animation.xml" path="Type[@FullName='Microsoft.Maui.Controls.Animation']/Docs/*" />
+	/// <summary>
+	/// Encapsulates an animation, a collection of functions that modify properties over a user-perceptible time period.
+	/// </summary>
 	public class Animation : BaseAnimation
 	{
 		bool _finishedTriggered;
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Animation.xml" path="//Member[@MemberName='.ctor'][1]/Docs/*" />
+		/// <summary>
+		/// Creates a new <see cref="Animation" /> object with default values.
+		/// </summary>
 		public Animation()
 		{
 			Easing = Easing.Linear;
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Animation.xml" path="//Member[@MemberName='.ctor']/Docs/*" />
+		/// <summary>
+		/// Creates a new <see cref = "Animation" /> object with the specified parameters.
+		/// </summary>
+		/// <param name="callback">An action that is called with successive animation values.</param>
+		/// <param name="start"> The fraction into the current animation at which to start the animation.</param>
+		/// <param name="end"> The fraction into the current animation at which to end the animation.</param>
+		/// <param name="easing"> The easing function to use to transision in, out, or in and out of the animation.</param>
```

Comment:
```suggestion
		/// <param name="easing"> The easing function to use to transition in, out, or in and out of the animation.</param>
```

Reply:
Ha I think I just copied these from Forms, so this has been a long time around :D